backend/src/utils/session.js

```
@@ -0,0 +1,52 @@
+/**
+ * retrieve userId from session
+ */
+export function getUserId(session) {
+   try {
+      const userId = session?.userId;
+      if (!userId) {
+         throw new ValidationError("User ID is required");
+      }
+      return userId;
+   } catch (error) {
+      throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)